### PR TITLE
feat: add gas price average min max

### PR DESCRIPTION
**Pull request by BelfordZ**
Adds the following to block view: 

1. gas price average 
2. gas price min
3. gas price max

image

Does not show when block has no transactions.

Also updates node, typescript and react build scripts

### CODE REVIEW

**Comment by shanejonas** [comment]
Linting issue

image

File: src/components/BlockView/BlockView.tsx [hunk 1]
```diff
@@ -134,7 +134,7 @@ function BlockView(props: any) {
       </Table>
 
       <TxList transactions={transactions} />
-    </ div >
+      </ div >
```
Comment (shanejonas) [inline src/components/BlockView/BlockView.tsx]: formatting issue

Comment (BelfordZ) [inline src/components/BlockView/BlockView.tsx:137]:
```suggestion
    </ div >
```

File: src/components/BlockView/BlockGasPrice.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <TableCell>{t("Min Gas Price")}</TableCell>
+        <TableCell>
+          {/* {hexToNumber(_.minBy(transactions, (t: any) => BigInt(t.gasPrice)).gasPrice)} */}
+        </TableCell>
+      </TableRow>
+
+      <TableRow>
+        <TableCell>{t("Max Gas Price")}</TableCell>
+        <TableCell>
+          {/* {hexToNumber(_.maxBy(transactions, (t: any) => BigInt(t.gasPrice)).gasPrice)} */}
```
Comment (shanejonas) [inline src/components/BlockView/BlockGasPrice.tsx:33]: commented?

File: src/App.tsx [hunk 3]
```diff
@@ -227,15 +237,15 @@ function App(props: any) {
                   selectedNetwork={selectedNetwork}
                 />
                 <LanguageMenu />
-                <Tooltip title={t("JSON-RPC API Documentation")}>
+                <Tooltip title={t("JSON-RPC API Documentation") as any}>
```
Comment (shanejonas) [inline src/App.tsx:240]:
```suggestion
                <Tooltip title={t("JSON-RPC API Documentation") as string}>
```

File: src/App.tsx [hunk 4]
```diff
@@ -227,15 +237,15 @@ function App(props: any) {
...
                   <IconButton
                     onClick={() =>
                       window.open("https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/etclabscore/ethereum-json-rpc-specification/master/openrpc.json") //tslint:disable-line
                     }>
                     <NotesIcon />
                   </IconButton>
                 </Tooltip>
-                <Tooltip title={t("Expedition Github")}>
+                <Tooltip title={t("Expedition Github") as any}>
```
Comment (shanejonas) [inline src/App.tsx:248]:
```suggestion
                <Tooltip title={t("Expedition Github") as string}>
```

File: src/App.tsx [hunk 5]
```diff
@@ -244,7 +254,7 @@ function App(props: any) {
                   </IconButton>
                 </Tooltip>
                 <ConfigurationMenu onChange={handleConfigurationChange} />
-                <Tooltip title={t("Toggle Dark Mode")}>
+                <Tooltip title={t("Toggle Dark Mode") as any}>
```
Comment (shanejonas) [inline src/App.tsx:257]:
```suggestion
                <Tooltip title={t("Toggle Dark Mode") as string}>
```

File: src/containers/NodeView.tsx [hunk 6]
```diff
@@ -11,10 +11,18 @@ interface IUrlParams {
...
-  const [erpc]: [EthereumJSONRPC] = useCoreGethStore();
+  const [erpc]: [EthereumJSONRPC, any] = useCoreGethStore();
   const [blockNumber] = useBlockNumber(erpc);
   const urlParams = useParams<IUrlParams>();
-  const blockNum = (urlParams && urlParams.number !== undefined) ? urlParams.number : blockNumber;
+
+  let blockNum = blockNumber;
+  if (urlParams && urlParams.number !== undefined) {
+    try {
+      blockNum = parseInt(urlParams.number)
```
Comment (shanejonas) [inline src/containers/NodeView.tsx:21]:
```suggestion
      blockNum = parseInt(urlParams.number, 10)
```

**Review by shanejonas — APPROVED** [review]

**Comment by shanejonas** [comment]
might as well get this in before it sits any longer

**Comment by shanejonas** [comment]
think we will need to do better bigint handling for minby/maxby

**Pull request merged**